ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -823,7 +1045,9 @@ ActivateSubScreen:
     .noRain
     
     ; Get the overlay value for this overworld area.
-    JSL.l ReadOverlayArray : CMP.w #$00FF : BEQ .normal
+    ; ReadOverlayArray 
+    LDA.b $8A : ASL : TAX
```

Comment:
Add a reason why we chose not to jump to ReadOverlayArray instead?

Reply:
I think it just needed to be read in 16 bit. the ReadOverlayArray function isn't vanilla so really this is just a slightly different way of doing the same thing. I'll remove the ReadOverlayArray part for clarity.